### PR TITLE
[EN-2082][EN-2083][EN-2084][EN-2085][EN-2086] Add Luna Toolbox

### PR DESCRIPTION
**Pull request by ilya-shablovsky**
Add Luna Toolbox.
Current features:
- Build Bridge lib and update dependencies in Luna
- Manage Bridge build version

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -212,3 +212,5 @@ Tests/**/App_Readme/
 # y no work?
 # !/Bridge/Bridge/Resources/.generated/System/Diagnostics/Debug/*
 /Bridge.Dev.Sandbox.sln
+Luna Toolbox/node_modules
```
Comment (MakeevAlexey) [inline .gitignore:215]: remove space:
LunaToolbox

File: Luna Toolbox/lib/cli.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const chalk = require('chalk');
+const { build, validateConfig, getConfig } = require('./jobs');
+const optionsParser = require('./optionsParser');
+
+function getTask(options) {
```
Comment (MakeevAlexey) [inline Luna Toolbox/lib/cli.js:5]: apply code styles

File: Luna Toolbox/lib/defines.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+			lunaCompiler: path.join(this.lunaPath, 'pipeline/templates/LunaCompiler'),
+			lunaCompilerSln: path.join(this.lunaPath, 'pipeline/templates/LunaCompiler/LunaCompiler.sln'),
+			lunaCompilerPackages: path.join(this.lunaPath, 'pipeline/templates/LunaCompiler/packages'),
+			nugetConfig: path.join(this.lunaPath, 'pipeline/templates/LunaCompiler/nuget.config'),
+			packagesToCleanUp: [ 'Bridge.Core', 'Bridge.Min' ],
+			csprojs: [
+				path.join(this.lunaPath, 'pipeline/templates/LunaCompiler/Scripts/Scripts.csproj'),
+				path.join(this.lunaPath, 'pipeline/templates/LunaCompiler/Scripts/Scripts.csproj.template'),
+				path.join(this.lunaPath, 'pipeline/templates/LunaCompiler/UnityEngine/UnityEngine.csproj'),
+				path.join(this.lunaPath, 'pipeline/templates/LunaCompiler/Vendor/com.unity.ui/1.2.1/1.2.1.csproj'),
```
Comment (MakeevAlexey) [inline Luna Toolbox/lib/defines.js:38]: This pathes can change - we need to correctly handle that issue

File: Luna Toolbox/lib/jobs/build.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
+const execa = require('execa');
+const Listr = require('listr');
+const path = require('path');
+const fs = require('fs');
+const replace = require('replace-in-file');
+const { Paths, Regex } = require('../defines');
+
+let paths;
+
+function build(options, config) {
```
Comment (MakeevAlexey) [inline Luna Toolbox/lib/jobs/build.js:10]: Please - write down in Confluence, how to extend your solution

**Review by MakeevAlexey — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ilya-shablovsky) [reply .gitignore]: Done.

File: Luna Toolbox/lib/cli.js (hunk 2 above)
Comment (MakeevAlexey) [reply Luna Toolbox/lib/cli.js]: and actually enable linter for that

File: Luna Toolbox/lib/defines.js (hunk 3 above)
Comment (ilya-shablovsky) [reply Luna Toolbox/lib/defines.js]: Just checked your concern, if any path is wrong - the utility will let you know with the message `No files match the pattern: [filepath]`

File: Luna Toolbox/lib/jobs/build.js (hunk 4 above)
Comment (ilya-shablovsky) [reply Luna Toolbox/lib/jobs/build.js]: Done.
https://lunalabsltd.atlassian.net/l/c/B3YS4954

File: Luna Toolbox/lib/cli.js (hunk 2 above)
Comment (ilya-shablovsky) [reply Luna Toolbox/lib/cli.js]: Done.

**Review by MakeevAlexey — APPROVED** [review]

**Pull request merged**